src/main/resources/templates/db_scripts/schema.json

```diff
@@ -34,12 +34,16 @@
     },
     {
       "tableName":"accounts",
-      "fromModuleVersion":"15.0",
+      "fromModuleVersion":"15.9.2",
```

Comment:
According to RMB doc, it looks like fromModuleVersion should be 15.10.0, otherwise we won't get this new index when upgrading from 15.9.2 to 15.10.0

Reply:
Thanks. I didn't really know what this was about. Updated it.